tests/alpaka/CMakeLists.txt

```diff
@@ -26,6 +25,7 @@ endif()
 
 traccc_add_test( alpaka
    ${TRACCC_ALPAKA_TEST_SOURCES}
+   test_cca.cpp
```

Comment:
Is there a reason for this change? And indeed for keeping `TRACCC_ALPAKA_TEST_SOURCES` around at all?

Reply:
`test_cca.cpp` is now a C++ rather than CUDA/HIP/SYCL executable. But indeed, no need to keep the variable any more.